spec/RecapSpec.js

```diff
@@ -133,6 +133,42 @@ describe('The Recap export module', function() {
     });
   });
 
+  describe('uploadDocket, uploadAttachmentMenu, uploadDocument', function() {
+    var existingFormData;
+    beforeEach(function() {
+      setupChromeSpy();
+      existingFormData = window.FormData;
+      window.FormData = FormDataFake;
+    });
+    
```

Comment:
This whitespace at the end of line 143 should be struck.

Reply:
Done